### ISSUE TITLE
Format drive in GPT manually

### ISSUE THREAD

**Issue by DrissiReda** [ventoy/Ventoy#279]
I imagine there is some future plan to set up the usb drive in GPT instead of MBR, but in the meantime I tried setting it up manually. 
This is how I did it:

- Create GPT partition table
- Create an exFat 1st partition
- Create an ESP 2nd partition (remaining 50Mb)
- Extracted ventoy.img into the 2nd partition.

Now I know I need to extract the core or boot img (not sure which). I just don't know where. Just to make sure that the steps I've taken already were right I booted into my usb and manually told the grub where to load, and it worked perfectly.

Can anyone tell me how can I make the grub find the right partition automatically? What to extract and where?

**Comment by candrapersada** [comment]
How to install Ventoy manually for 1nd and 2nd partition without remove partition 3 on the right?
![image](https://user-images.githubusercontent.com/36833664/86306270-040a4e80-bc3e-11ea-8154-bd2e74dd44ff.png)

**Comment by ventoy** [comment]
@DrissiReda 
There is something related to MBR in current release, so you have to speficy the right partition manually.
It will be OK in the next release.

**Comment by ventoy** [comment]
@candrapersada 
I think it's more simple to backup the data first and install Ventoy with reserver space (just the of space E: ).
After install, you can create the 3rd partition, format it in NTFS and copy the backup data into it.

**Comment by DrissiReda** [comment]
@candrapersada that is not at all what I need, I don't have any issue doing what you suggested. I need to set the ventoy drive's partition table as GPT instead of MBR, that's all. 

@ventoy you mean the bootloader is being written in the first 2048 sectors along with MBR?

**Comment by ventoy** [comment]
@DrissiReda  
No,  the `root` variable was set to (hdX, msdos2)  by default and this is only for MBR.  
I will modify it in the next release.

**Comment by DrissiReda** [comment]
Can't I manually change the root and prefix variables on my drive? I'm using linux.

**Comment by ventoy** [comment]
No, it's embedded into the grub image.

**Comment by candrapersada** [comment]
can install windows 10 from GPT to MBR?

**Comment by DrissiReda** [comment]
depends on your motherboard firmware, but most UEFI boards support legacy.  So they can boot from both MBR and GPT drives.
Using a GPT usb key with windows 10 iso (not ventoy cuz like it says here it doesn't support GPT) on a drive with an MBR partition table, because the two aren't related.

**Comment by ventoy** [comment]
This is an old issue. I will close it now.
If you still have problem with latest Ventoy release, you can reopen the issue.
